packages/platform-sdk-nano/src/services/peer-host.ts

```diff
@@ -0,0 +1,10 @@
+import { Coins } from "@arkecosystem/platform-sdk";
+import { Arr } from "@arkecosystem/platform-sdk-support";
+
+export const getPeerFromConfig = (config: Coins.Config): string => {
```

Comment:
I see this function repeated in many places, would it make sense to factor it in some library package?

Reply:
https://github.com/ArkEcosystem/platform-sdk/pull/1542